### DISCUSSION TITLE
Failed to compile ARDESPOT

### DISCUSSION THREAD

**Discussion by Etwari3** [JuliaPOMDP/POMDPs.jl#470] [General]
Hello,

I tried to test ARDESPOT and I keep getting this error.

Failed to precompile ARDESPOT [d96c9ae4-3372-47d5-8a88-316ae77be8cf]

Stacktrace:
 [1] error(s::String)
   @ Base .\error.jl:35
 [2] compilecache(pkg::Base.PkgId, path::String, internal_stderr::IO, internal_stdout::IO, keep_loaded_modules::Bool)
   @ Base .\loading.jl:1707
 [3] compilecache
   @ .\loading.jl:1651 [inlined]
 [4] _require(pkg::Base.PkgId)
   @ Base .\loading.jl:1337
 [5] _require_prelocked(uuidkey::Base.PkgId)
   @ Base .\loading.jl:1200
 [6] macro expansion
   @ .\loading.jl:1180 [inlined]
 [7] macro expansion
   @ .\lock.jl:223 [inlined]
 [8] require(into::Module, mod::Symbol)
   @ Base .\loading.jl:1144

Can you please help?

Thank you.

**Comment by rejuvyesh** [comment]
Can you try in a new environment, just to be sure:

```julia
]
pkg> activate --temp
pkg> add ARDESPOT
using ARDESPOT
```
I couldn't reproduce your error but I don't have a Windows machine at the moment.

**Reply by Etwari3** [reply]
I am using Jupyter notebook. Do I need to use julia app?

**Reply by zsunberg** [reply]
I think you can do it from a notebook with
```julia
Pkg.activate(; temp=true)
Pkg.add("ARDESPOT")
using ARDESPOT
```

**Reply by Etwari3** [reply]
Thank you, zsunberg. That has solved the issue.
